pkg/app/instances/docker.go

```diff
@@ -381,6 +385,17 @@ func (m *DockerInstanceManager) createDockerContainer(ctx context.Context, user
 		},
 		Privileged: true,
 	}
+	if m.Config.Docker.GpuManufacturer == "nvidia" {
+		hostConfig.Resources = container.Resources{
+			DeviceRequests: []container.DeviceRequest{
+				{
+					Count:        -1,
+					Capabilities: [][]string{{"gpu"}},
+				},
+			},
+		}
+		hostConfig.Runtime = "nvidia"
+	}
```

Comment:
I think we'd better raise an exception if GpuManufacturer isn't empty, but GpuManufacturer is not supported one to avoid a user's mistake